### ISSUE TITLE
주제 선정

### ISSUE THREAD

**Issue by JewooJun** [OSS2023TP/project#2]
저희 주제는 'AI model 개발 일정 관리'로 결정지으면 될까요?
AI model 개발 일정 관리를 하는 것인데 현재 Accuracy와 DueDate, 완성도, 책임자, 기타사항 기록 할 수 있도록 하면 좋을 것 같습니다.

**Comment by amin7942** [comment]
네 AI model 개발 일정관리로 하면 될것같습니다. 저도 그 기록 할 수 있으면 좋겠습니다. 이전에 crud한 것 처럼 날짜순으로 데이터를 보여주고 그 정보로 검색을 하게끔 만들면 될 것 같습니다. 이후에 개선사항이 생기면 그때가서 기능을 추가하면 좋겠습니다

**Comment by JewooJun** [comment]
네 좋습니다 그럼 주제는 그렇게 하도록 하고 기능 이야기를 해봅시다
